types/nightwatch/test/nightwatch-tests.ts

```diff
@@ -73,6 +73,21 @@ const testGeneral: NightwatchTests = {
     'test user defined globals': () => {
         browser.url(`http://${browser.globals.username}:${browser.globals.password}@example.com`).end();
     },
+    
+    'Test assertions on page': () => {
```

Comment:
This test already existed in the file lower down. My PR comment was suggesting you edit the existing test with this name to include the .moveToElement line. Please remove this duplicate test block and add it to the one with the same name lower down and add `.moveToElement('@searchBar', 1, 1)` in the equivalent spot. Thanks again.